### PR TITLE
superseded phone number banner

### PR DESCRIPTION
**Pull request by pzduniak**

### CODE REVIEW

File: shared/settings/account/container.tsx [hunk 1]
```diff
@@ -8,16 +8,38 @@ import AccountSettings from '.'
...
-const mapStateToProps = (state: TypedState) => ({
-  _emails: state.settings.email.emails,
-  _phones: state.settings.phoneNumbers.phones,
-  addedEmail: state.settings.email.addedEmail,
-  bootstrapDone: state.settings.email.emails !== null && state.settings.phoneNumbers.phones !== null,
-  hasPassword: !state.settings.password.randomPW,
-  waiting: anyWaiting(state, Constants.loadSettingsWaitingKey),
-})
+const mapStateToProps = (state: TypedState) => {
+  const phoneNumberRecord =
```
Comment (jakob223) [inline shared/settings/account/container.tsx]: can this variable name include the fact that it's superseded?

File: shared/settings/account/index.tsx [hunk 2]
```diff
@@ -11,6 +11,8 @@ export type Props = {
   addedEmail: string | null
   contactKeys: I.List<string>
   hasPassword: boolean
+  supersededPhoneNumber?: ContactRowProps
```
Comment (jakob223) [inline shared/settings/account/index.tsx]: This doesn't really need to be a full `ContactRowProps`, does it? Can you use only the one or two fiels of that you actually need?

File: shared/settings/account/container.tsx [hunk 3]
```diff
@@ -8,16 +8,38 @@ import AccountSettings from '.'
...
+
+  return {
+    _emails: state.settings.email.emails,
+    _phones: state.settings.phoneNumbers.phones,
+    _supersededPhoneNumberKey: phoneNumberRecord && phoneNumberRecord.e164,
+    addedEmail: state.settings.email.addedEmail,
+    bootstrapDone: state.settings.email.emails !== null && state.settings.phoneNumbers.phones !== null,
+    hasPassword: !state.settings.password.randomPW,
+    supersededPhoneNumber: phoneNumberRecord
+      ? {
```
Comment (jakob223) [inline shared/settings/account/container.tsx]: can you cut out those of these fields you don't need?

Comment (pzduniak) [reply shared/settings/account/container.tsx]: i just realized that after all my changes i just needed 1 field

**Pull request merged**